.github/workflows/binaries-partial-release.yml

```diff
@@ -20,19 +31,37 @@ on:
 
 jobs:
   release:
-    name: Release CLI
+    name: Release binaries
```

Comment:
same remark on separate workflows, just use two different jobs imho, easier to manage than 10 `ifs` imho